rideshare.rb

```diff
@@ -0,0 +1,151 @@
+# RIDESHARE
+# Calculates statistics of rideshare data.
+
+drivers = {
+  DR0001: [
+    {
+      "3rd Feb 2016": [
```

Comment:
One note about a confusing point of Ruby.  Even if you use quotes if you use the `"key": value` syntax instead of the `"key" => value` the key will wind up a symbol.

Note the `:` before the quotes in the output.
```ruby
irb(main):001:0> {"3rd Feb 2016": []}
=> {:"3rd Feb 2016"=>[]}
```

(Yes, symbols can have spaces, you just need to use quotes after the `:`.)